test/visual-diff/visual-tests

```diff
@@ -683,6 +683,26 @@
         ".i-amphtml-story-loaded",
       ],
     },
+    {
+      "flaky": true, // Skip this until shadow DOM the embeds support shadow DOM
```

Comment:
Not sure I understand why the visual diff tests aren't working since we already support components in the shadow DOM like the bookend?

Reply:
For the core stories framework, we're actually not testing the shadow DOM code.  When we create new shadow roots, we use a utility; that utility simply does not create a shadow root in test code.

https://github.com/ampproject/amphtml/blob/f4fe32a27b8febeb457b7ef1764d1fa55380c42e/extensions/amp-story/1.0/utils.js#L102-L112

Since this code doesn't live in the `amp-story` extension, we don't use that util.  We can use the same high-level approach, but to do so, we have to rewrite a bunch of things (all the styles break), so I figured it would be better to do as a separate PR.

Reply:
Got it, thanks!